java/java.disco/nbproject/project.xml

```diff
@@ -82,6 +82,15 @@
                         <specification-version>1.18</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
```

Comment:
This is the dependency I'm missing in `db.dataview`.

Reply:
Dependency added and re-tested everything.